packages/functional-tests/tests/oauth/signin.spec.ts

```diff
@@ -186,7 +186,11 @@ test.describe('severity-1 #smoke', () => {
       page,
       pages: { login, relier, settings, deleteAccount },
       emails,
-    }) => {
+    }, { project }) => {
+      test.fixme(
```

Comment:
Could you please add the Jira ticket reference here in the comments too? Thanks!